### ISSUE TITLE
 the audio file is not playing

### ISSUE THREAD

**Issue by jiangjiatao** [Dreaming381/Latios-Framework-Documentation#5]
I followed this documentation but the audio file is not playing

![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/0fa89f6f-6d79-4778-aa51-aaf90bcb95af)

**Comment by Dreaming381** [comment]
I have a few questions.

First, which Latios Framework version and Unity Editor version are you using?
Second, can you please share screenshots of the inspector for both the Myri Audio Source and the Myri Audio Listener?

**Comment by jiangjiatao** [comment]
Is Bootstrap a must-have? I can't find the option in the menu。

![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/e9d0577c-0a1d-4263-8432-0922c1aa6919)
![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/645acbed-2783-4dcc-afbe-78b9c80a9636)
![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/08326680-4254-4594-9f9e-7fb47c85536f)
![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/8fa83261-0f07-462d-81c6-52612d46a349)
![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/8fe77636-eb24-4946-a867-8674091f6891)
![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/1e2bc97d-d975-42ef-bf81-d44f0c159b01)

**Comment by jiangjiatao** [comment]
I need your help very much

Unity Editor version

![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/c4237793-c1b9-4e34-a8da-c246d53fec48)

**Comment by Dreaming381** [comment]
First off, Myri only works correctly in Unity 2022.3.13f1 or newer. Versions prior to that rarely worked correctly with imported audio clips due to a bug on Unity's side.

Second, the bootstrap creation menu is not in the Component menu, but rather the Assets menu. The bootstrap is a hard requirement.

**Comment by jiangjiatao** [comment]
It can be used correctly, thank you

**Comment by jiangjiatao** [comment]
my Unity Editor version is 2022.3.10f1,  are these errors caused by the unity version?

![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/6b33c5d3-c44c-43e7-9a4c-46b793a0c262)
![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/95c3096a-03ce-4a55-9b00-131fd833712b)
![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/0f75e977-713a-4c23-83b2-1b1276ee553d)

**Comment by Dreaming381** [comment]
I do not believe those errors are related. Typically, Myri errors with the editor version will happen during subscene baking and cause Unity to crash.

**Comment by jiangjiatao** [comment]
There are related error reports

![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/18566483/256c3282-5101-4904-9776-3ce5e012d25d)

**Comment by Dreaming381** [comment]
That's just a custom `IRateManager` that monitors for Latios Framework systems and performs extra operations. For other systems, it is just doing what `ComponentSystemGroup` normally does. You will see these show up in every callstack that the Latios Framework is installed in. The actual issues are in the invoked systems themselves. In this case, the issue is that your `EntityCommandBuffer` is trying to remove a component on an entity that does not have the component in the first place. Nothing in the Latios Framework even uses `EntityCommandBufferSystem`, so this is a bug in your code.

The Latios Framework has a special feature that you can enable in the bootstrap file via `world.zeroToleranceForExceptions= true;`
This will cause the world to pause and not try to run any more systems once one of them throws an exception, hiding exceptions that were just side-effects of things being broken to start with.

**Comment by jiangjiatao** [comment]
The problem I'm encountering now is that if I don't add LatiosBakingBootstrap, there won't be any problems. But after adding LatiosBakingBootstrap, running unity and entering my game scene, the above errors will be reported. Can you give me some guidance? I just I started studying dots, but my proficiency is not high.

**Comment by Dreaming381** [comment]
I'm sorry. I think I know what your issue might be.

From your screenshots, it appears you have Unity Physics installed. If you simply installed the Latios Framework and created the _Standard - Injection Workflow_ bootstrap, you unfortunately created an incompatibility. Usually the Kinemation renderer is the first to catch this, but (assuming I am right about this) it appears that wasn't the case for you.

My guess is that you are missing the LATIOS_TRANSFORMS_UNITY scripting define. If you intend to use QVVS Transforms, you cannot use Unity Physics without modifying the package.
